### DISCUSSION TITLE
Powergrid Icons not being displayed properly

### DISCUSSION THREAD

**Discussion by philozofer** [Power-Components/livewire-powergrid#1292] [Q&A] [answered]
Hi, I'd like to know if there is a way to fix the "Magnifiy Glass Icon" in the _**search bar**_  && also the "Chrevron icon" for _**show per page**_ to be displayed properly ? 

![image](https://github.com/Power-Components/livewire-powergrid/assets/9042494/766b6fb3-507e-4181-bfc4-9028d4d31b84)

Code that renders these two elements through a function : 
```
 public function setUp(): array
    {
        $this->showCheckBox();
        return [
            Exportable::make('export')
                ->striped()
                ->type(Exportable::TYPE_XLS, Exportable::TYPE_CSV),
            Header::make()->showSearchInput(), // Search Input Bar
            Footer::make()
                ->showPerPage() // Results per page
                ->showRecordCount(),
        ];
    }
```

I did include  the powergrid livewire component into a modal, could that cause conflicts of some sort ?
```
 <div class="sticky bottom-0 right-8">
        <x-button x-on:click="$openModal('staffsModal')" class="bg-orange-600 text-xs text-white uppercase rounded-md border hover:text-white hover:bg-orange-300">Click ME</x-button>
    </div>

    <x-modal name="staffsModal" maxWidth="4xl">
        <x-card title="Staffs">
            <livewire:staffTable />

            <x-slot name="footer" class="flex justify-end gap-x-4">
                <x-button flat label="Cancel" x-on:click="close" />

                <x-button primary label="I Agree" wire:click="agree" />
            </x-slot>
        </x-card>
    </x-modal>

```

Same on a regular page: 
![image](https://github.com/Power-Components/livewire-powergrid/assets/9042494/89f12647-d114-4c86-8195-be4b8110c207)

**Comment by tumut** [comment]
I noticed the same problem.

**Comment by luanfreitasdev** [comment]
Please read the docs: https://livewire-powergrid.com/get-started/configure.html#tailwind-settings

**Reply by luanfreitasdev** [reply, marked as answer]
Fix this conflict and will be released in version 5.2

https://github.com/Power-Components/powergrid-doc/pull/88

**Reply by kwasii1** [reply]
I still have the problem
